niaiddata.org/portal/gitops.json

```diff
@@ -192,65 +189,154 @@
+      { "field": "species", "name": "Genus Species" },
+      { "field": "submitter_id", "name": "Submitter ID" },
+      { "field": "race", "name": "Race" },
+      { "field": "hiv_status", "name": "HIV Status" },
+      { "field": "study_submitter_id", "name": "Study"},
+      { "field": "frstdthd", "name": "Year of Death" },
+      { "field": "arthxbase", "name": "ART Use Prior to Baseline"},
+      { "field": "bshbvstat", "name": "Baseline HBV Sero-status"},
+      { "field": "bshcvstat", "name": "Baseline HCV Sero-status"},
+      { "field": "cd4nadir", "name": "CD4 Nadir Prior to HAART"},
+      { "field": "status", "name": "Summarized HIV Sero-status"},
+      {"field": "project_id", "name": "Project ID"},
+      {"field": "frstcncrd", "name": "First Confirmed Cancer Year"},
+      {"field": "frstdmd", "name": "First Visit Year with Diabetes"},
+      {"field": "frstdmmd", "name": "First Visit Year with All Necessary Components to Determine Diabetes"},
+      {"field": "frsthtnd", "name": "First Visit Year with Hypertension"},
+      {"field": "frsthtnmd", "name": "First Visit Year with All Necessary Components to Determine Hypertension"},
+      {"field": "fcd4lowd", "name": "First Year Seen CD4N < 200 or CD4% < 14"},
+      {"field": "fposdate", "name": "First Year Seen Seropositive"},
+      {"field": "frstaidd", "name": "First Reported AIDS Year"},
+      {"field": "lastafrd", "name": "Last Reported AIDS Free Year"},
+      {"field": "lastcond", "name": "Year of Last Study Visit Attended"},
+      {"field": "lastcontact", "name": "Last Year of Contact"},
+      {"field": "lcd4higd", "name": "Last Year Seen with CD4N >= 200 and CD4% >= 14"},
+      {"field": "lnegdate", "name": "Last Year Seen Seronegative"}
     ],
     "filterConfig": {
       "tabs": [{
+        "title": "Resource",
+        "fields": ["dataset", "data_format", "data_type"]
+      },
+      {
         "title": "Subject",
-        "fields": ["ethnicity", "gender", "species"]
+        "fields": ["ethnicity", "gender", "species", "race", "experimental_strategies"]
       },
       {
-        "title": "Resource",
-        "fields": ["dataset", "research_focus"]
+        "title": "Diagnosis",
+        "fields": [
+          "arthxbase",
+          "bshbvstat",
+          "bshcvstat",
+          "cd4nadir",
+          "status",
+          "hiv_status"
+        ]
+      },
+      {
+        "title": "Comorbidity",
+        "fields": [
+          "frstcncrd",
+          "frstdmd",
+          "frstdmmd",
+          "frsthtnd",
+          "frsthtnmd"
+        ]
+      }, {
+        "title": "HIV History",
+        "fields": [
+          "cd4nadir",
+          "fcd4lowd",
+          "fposdate",
+          "frstaidd",
+          "lastafrd",
+          "lastcond",
+          "lastcontact",
+          "lcd4higd",
+          "lnegdate",
+          "status"
+        ]
+      },
+      {
+        "title": "Drug Resistance",
+        "fields": [
+          "amikacin_res_phenotype",
```

Comment:
Can we add a field mapping for these properties so that they just read `drug name phenotype' ie. 'Amikacin Phenotype'